tests/python/contrib/test_tensorrt.py

```diff
@@ -474,14 +474,22 @@ def get_graph(x_shape=(1, 16), k_shape=(32, 16)):
 
 
 def test_batch_matmul():
-    def get_graph(x_shape=(12, 128, 64), y_shape=(12, 128, 64)):
+    def get_graph(x_shape=(12, 128, 64), y_shape=(12, 64, 128), transa=False, transb=True):
         x = relay.var("x", shape=(x_shape), dtype="float32")
         y = relay.var("y", shape=(y_shape), dtype="float32")
-        out = relay.nn.batch_matmul(x, y)
+        out = relay.nn.batch_matmul(
+            relay.transpose(x, [0, 2, 1]) if transa else x,
```

Comment:
I don't think you need these `relay.transpose` on the inputs to test functionality of transa/transb args.

Reply:
Good point, I've changed to using x/y_shape instead.